### ISSUE TITLE
[Wrong value] Wrong device metric

### ISSUE THREAD

**Issue by smith-prnd** [material-components/material-components-android#42]
It may not be related to this repository. But I do not know where to report it, so I leave it here.

[This page](https://material.io/devices/) has an invalid value.
This page says that the size of the galaxy s8 is 360x740dp. But actually it is 441x906dp

Please fix it! Thanks!

**Comment by miguelandres** [comment]
Scott, do you know who the owner for that page is?

**Comment by shyndman** [comment]
Yeah, I can get this fixed.

@smith-prnd, can you please point me to something confirming that resolution?

**Comment by smith-prnd** [comment]
@shyndman 
@miguelandres 

The spec of Galaxy S8 is following as below.

![2017-09-20 11 45 02](https://user-images.githubusercontent.com/30643078/30624585-2ef01d20-9df9-11e7-89d1-8c16e68b31ae.png)

It has 2960x1440 px and 571 ppi. (dpi == ppi)
So we can calculate the width and height dp.

```
dp = px / (dpi / 160f)

widthDp = 1440 / (571 / 160) = 403dp
heightDp = 2960 / ( 571 / 160) = 829dp
```
(The 441x906dp that I said in the question 
seems to have miscalculated. It's size of Galaxy S8+. sorry.)

I tested it in Galaxy S8+ (it has similar width GalaxyS8)

![20170920_115342_1024](https://user-images.githubusercontent.com/30643078/30624904-ee998246-9dfa-11e7-91db-e41e32950bdb.jpg)


To sum up it, 
The WidthxHeight (DP) on the page below should be changed to blow.

```
Galaxy S8        -> 1440x2960px, 403x829dp (571 ppi)
Galaxy S8+       -> 1440x2960px, 441x906dp (522 ppi)
```

![2017-09-20 12 01 26](https://user-images.githubusercontent.com/30643078/30624985-780b0298-9dfb-11e7-841b-8975473e4bfb.png)

**Comment by smith-prnd** [comment]
@shyndman 
@miguelandres 
is there any changes??

**Comment by smith-prnd** [comment]
@shyndman 
@miguelandres 
Not yet changed....
is there any problem?
